MSAL/src/MSALPublicClientApplication.m

```diff
@@ -1511,6 +1511,12 @@ + (NSOrderedSet *)defaultOIDCScopes
                                                MSID_OAUTH2_SCOPE_OFFLINE_ACCESS_VALUE, nil];
 }
 
++ (NSString *)sdkVersion
+{
+    return [[NSBundle bundleForClass:MSALPublicClientApplication.class] objectForInfoDictionaryKey:@"CFBundleShortVersionString"];
```

Comment:
shouldn't we read it from MSAL_internal instead? That is how MSAL SDK version is always reported.

Reply:
Thanks for pointing that out. Applied a fix